example/lib/screens/message_center.dart

```diff
@@ -23,17 +40,18 @@ class MessageCenter extends StatelessWidget {
                   .of(context)
                   .showSnackBar(SnackBar(content: Text("Message \"${message.messageId}\" removed")));
               messages.remove(message);
-              _airshipBloc.messageRemovedSink.add(message);
+              Airship.deleteInboxMessage(message);
+              setState(() {});
             },
             // Add stream to check isRead
             child: ListTile(
               title: message.isRead ? Text('${message.title}') : Text('${message.title}', style:TextStyle(fontWeight: FontWeight.bold)),
               subtitle: Text('${message.sentDate}'),
-              leading: Icon(message.isRead ? Icons.markunread : Icons.check_circle) ,
+              leading: Icon(message.isRead ? Icons.check_circle: Icons.markunread),
```

Comment:
Is this right? It looks like you will display `check_circle` when the message `isRead`, and `markunread` when the message is not `isRead`, which I'm interpreting as `isUnread`. It seems like you'd want it the way it was before.

Reply:
If markunread doesn't imply that the message is unread, what does it imply?

Reply:
This behavior is correct. Check circle appears when the message isRead, markunread (an unopened envelope icon) appears otherwise. Somehow I had this swapped last time. You can confirm this functionality in the video.

Reply:
:cool: